### PR TITLE
Update making-release.md

### PR DESCRIPTION
**Pull request by milleruntime**

### CODE REVIEW

File: contributor/making-release.md [hunk 1]
```diff
@@ -79,7 +80,8 @@ will make it publicly available for other members to inspect.
 
 At this point, you should have a closed repository that's ready to vote on. Send a message to [the dev
 list](mailto:dev@accumulo.apache.org) and get the ball rolling. Developers should test and verify the
-release candidate on their own. Accumulo has a guide for [verifying releases][verify].
+release candidate on their own. Accumulo has a guide for [verifying releases][verify].  The provided 
+script assemble/build.sh can be used to generate a release VOTE email using the --
```
Comment (dlmarion) [inline contributor/making-release.md:84]: Did this get cut off?

Comment (milleruntime) [reply contributor/making-release.md:84]: It is unfinished.  I marked this as a draft since its still WIP.  I have had it open in my browser with edits as I have done the release but didn't want to lose the changes.

File: contributor/making-release.md [hunk 2]
```diff
@@ -19,11 +19,13 @@ Follow these steps to make a release of Apache Accumulo.
 
 There are number of things that are required before attempting to build a release.
 
-1. Use gpg-agent, and be sure to increase the gpg-agent cache timeout (via .gnupg/gpg-agent.conf) to ensure that the agent doesn't require re-authentication mid-build, as it will cause things to fail. For example, you can add `default-cache-ttl 6000` to increase the timeout from the default of 10 minutes to over an hour. If you do not have a GPG key, reference the very thorough [ASF release signing documentation][1].
+1. Use gpg-agent, and be sure to increase the gpg-agent cache timeout (via .gnupg/gpg-agent.conf) to ensure that the agent doesn't require re-authentication mid-build, as it will cause things to fail. For example, you can add `default-cache-ttl 6000` to increase the timeout from the default of 10 minutes to over an hour. If you do not have a GPG key, reference the very thorough [ASF release signing documentation][1].  Once you are able to sign a release, add your gpg key to the [KEYS file][keys-file] (instructions are at the top of the file).
```
Comment (ctubbsii) [inline contributor/making-release.md:22]: The information about the KEYS file should be a separate bullet point.

File: contributor/making-release.md [hunk 3]
```diff
@@ -50,8 +52,7 @@ Before creating a release candidate, all open issues with a fix version of the r
 
 You should use the provided script assemble/build.sh to create the release candidate. This script is
 desirable as it activates all necessary maven profiles in addition to verifying that certain preconditions
-are met, like RPM signing availability and the ability to sign files using GPG. The --test option can
-be used as a dry run for creating a release candidate. The --create-release-candidate option should be 
+are met, like RPM signing availability and the ability to sign files using GPG. The --create-release-candidate option is
```
Comment (ctubbsii) [inline contributor/making-release.md:55]: We can drop the information RPM signing info, as we don't build RPMs anymore.

File: contributor/making-release.md [hunk 4]
```diff
@@ -35,7 +37,7 @@ Before creating a release candidate, all open issues with a fix version of the r
 
 **TL;DR**
 
-* `./assemble/build.sh --create-release-candidate` to make the release candidate
+* `./assemble/build.sh --create-release-candidate -DskipTests -Dspotbugs.skip` to make the release candidate
```
[the remark this answers is not in the document]
Comment (milleruntime) [reply contributor/making-release.md:40]: Sounds good.  I am confused about the commands after this though and before the **VOTE**... 
```
* `git tag $version $version-rcN` to create an RC tag from the actual tag
* `git tag -d $version` make sure you don't accidentally push a "release" tag
* `git push origin $version-rcN` push the RC tag
* `git checkout -b $version-rcN-branch` save off the branch from the Maven release plugin
```
Don't we want to wait and tag AFTER the vote is successful?

[the remark this answers is not in the document]
Comment (ctubbsii) [reply contributor/making-release.md:40]: Yes. We should not tag until after the vote. I did not write these instructions, and the script I made should delete the tag that the release plugin automatically creates, and creates a branch instead.

File: contributor/making-release.md [hunk 5]
```diff
@@ -199,15 +205,18 @@ if this is the latest release of Accumulo.
 Some good references that explain a few things:
 
 - [Christopher talks about making releases][3]
```
Comment (ctubbsii) [inline contributor/making-release.md:207]: Oh wow, I just clicked on this link and this is me talking about releases when we were using SVN. We haven't done that in a *VERY* long time. This information is at least 7 years old, and almost entirely irrelevant to how we release today.

**Comment by milleruntime** [comment]
@dlmarion This had been in draft but I think its ready to merge if you want another look.

**Comment by dlmarion** [comment]
+1 looks good
